epoxy-paging/src/main/java/com/airbnb/epoxy/paging/PagingEpoxyController.java

```diff
@@ -177,6 +177,24 @@ public void setList(@Nullable PagedList<T> list) {
     updatePagedListSnapshot();
   }
 
+  /**
+     * Returns the list currently being displayed by the EpoxyController.
+     *
+     * @return The list currently being displayed.
+     */
+  public List<T> getCurrentList() {
+    return this.list == null ? Collections.<T>emptyList() : this.list;
+  }
+
+  /**
+     * Returns the pagedList currently being displayed by the EpoxyController.
+     *
+     * @return The pagedList currently being displayed.
+     */
+  public PagedList<T> getPagedList() {
+    return this.pagedList == null ? (PagedList<T>) Collections.<T>emptyList() : this.pagedList;
```

Comment:
You can't just cast the list to a PagedList, that will crash. I think we need to return null in this case, it isn't possible to create our own empty paged list

Reply:
yes, null check here is unnecessary.